nemo_text_processing/text_normalization/en/taggers/roman.py

```diff
@@ -60,7 +60,6 @@ def _load_roman(file: str):
         graph = pynini.compose(pynini.closure(NEMO_CHAR, 2), graph) | pynutil.add_weight(
             pynini.compose(NEMO_CHAR, graph), 101
         )
-        graph = graph.optimize() + pynini.closure(pynutil.delete("."), 0, 1)
```

Comment:
I don't know about this deleting and it is not in main. Is this an accidental deletion or not? if so, do you plan to add it to main?